squad/frontend/templates/squad/_builds_table.html

```diff
@@ -6,6 +6,7 @@
 <a href="{% build_url build %}">
 <div class="row row-bordered build">
     <div class="col-md-2 col-sm-2">
+        {% if not build.status.finished %}<i class="glyphicon glyphicon-refresh"></i>{% endif %}
```

Comment:
I think there should be an alt/title attribute with a description, for accessibility and also so that there is some sort of information about the "refresh" icon means.

also, the new bootstrap version dropped the glyphicons, so maybe we should be using FontAwesome here instead.

Reply:
It would be nice to see % completed or number of jobs completed vs outstanding (if either are easy/possible). Maybe in the alt/title if there's no room for it on the page itself.

Reply:
I would go with glyphicons for now. I'll create an issue to migrate to bootstrap 4 which should cover replacing all glyphicons to FA.

The number of test jobs is a bit harder to get in this view. We don't cache the number of test jobs in ProjectStatus so this would have to be queried 'live'. And it's at least 1 'count' query per build. This would be a significant performance hit.